nicegui/elements/json_editor.py

```diff
@@ -1,8 +1,11 @@
-from typing import Callable, Dict, Optional
+from typing import Any, Callable, Dict, Optional
+
+from typing_extensions import Self
 
 from ..awaitable_response import AwaitableResponse
 from ..element import Element
-from ..events import GenericEventArguments, JsonEditorChangeEventArguments, JsonEditorSelectEventArguments, handle_event
+from ..events import (GenericEventArguments, JsonEditorChangeEventArguments,
```

Comment:
this formatting was not intentional, but it did happen when i ran autopep8, and it doesn't seem _wrong_, either - the line was previously 121 characters long, which is over the 120 character limit?

Reply:
Yeah, that's strange. See https://github.com/zauberzeug/nicegui/issues/2750.